test/unit/docsify.test.js

```diff
@@ -0,0 +1,133 @@
+/* global before after */
+/* eslint-disable no-global-assign */
+require = require('esm')(module /* , options */);
+
+const http = require('http');
+const handler = require('serve-handler');
+const { expect } = require('chai');
+const { initJSDOM } = require('../_helper');
+
+const docsifySite = 'http://127.0.0.1:3000';
+
+const markup = /* html */ `<!DOCTYPE html>
+  <html>
+    <head></head>
+    <body>
+      <div id="app"></div>
+      <script src="/lib/docsify.js"></script>
+    </body>
+  </html>
+`;
+
+/** @type {ReturnType<typeof http.createServer>} */
+let server;
+
+describe('Docsify public API', () => {
+  before(async () => {
+    server = http.createServer((request, response) => {
+      // You pass two more arguments for config and middleware
+      // More details here: https://github.com/zeit/serve-handler#options
+      return handler(request, response);
+    });
```

Comment:
This creates an HTTP server, when we run JSDOM it will fetch the `<script src="/lib/docsify.js"` from this local server.